core/src/main/java/org/keycloak/representations/AuthorizationToken.java

```diff
@@ -0,0 +1,11 @@
+package org.keycloak.representations;
+
+import org.keycloak.TokenCategory;
+
+public class AuthorizationToken extends JsonWebToken{
```

Comment:
Is it possible to rename this class to `AuthorizationResponseToken` and rename category to `AUTHORIZATION_RESPONSE`? The reason is, that term "Authorization" is already overloaded and is used in Keycloak in various other contexts. So just to make this a bit more clear.

Reply:
Sure